services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/application/AlluUpdateService.kt

```diff
@@ -21,7 +21,7 @@ class AlluUpdateService(
 
     internal val lockName = "alluHistoryUpdate"
 
-    @Scheduled(fixedDelay = 1000 * 60, initialDelay = 1000 * 60)
+    @Scheduled(fixedDelay = 1000 * 60, initialDelay = 1000000 * 60)
```

Comment:
Why so long initial delay? Could a constant improve some readability here? Something like:

```
private const val X_MINUTES =  value here
private const val Y_MINUTES = value here

@Service
class AlluUpdateService(
...

    @Scheduled(fixedDelay = Y_MINUTES, initialDelay = X_MINUTES)
    fun checkApplicationStatuses() {
    
```

Where x and y are the duration.

Reply:
I changed the initial delay to get clearner logs at some in the development and accidentally committed it. I reverted this change.

I think it would be a good idea to read the values from the configuration, so they can be overridden. Outside this PR, though.